src/key-value-pairs/internal.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.css.js';
+import { KeyValuePairsProps } from './interfaces';
+import { InfoLinkLabelContext } from '../internal/context/info-link-label-context';
+import ColumnLayout from '../column-layout/internal';
+import Box from '../box/internal';
+
+export { KeyValuePairsProps };
+
+const InternalKeyValuePair = ({ label, info, value }: KeyValuePairsProps.Pair) => (
+  <>
+    <dt className={styles.term}>
+      <label className={styles['key-label']}>{label}</label>
+      <InfoLinkLabelContext.Provider value={label}>
+        {info && <span className={styles.info}>{info}</span>}
+      </InfoLinkLabelContext.Provider>
+    </dt>
+    <dd className={styles.detail}>{value}</dd>
+  </>
+);
+
+const InternalKeyValuePairGroup = ({ label, value }: { label?: React.ReactNode; value: React.ReactNode }) => (
+  <>
+    {label && <dt className={styles['group-title']}>{label}</dt>}
+    <dd className={styles.detail}>{value}</dd>
+  </>
+);
+
+const InternalKeyValuePairs = React.forwardRef(
+  ({ columns = 1, items, className, ...rest }: KeyValuePairsProps, ref: React.Ref<HTMLDivElement>) => {
```

Comment:
If you set "columns = 1" in the index file, we'll be able to see the default value 1 in API description.

Reply:
updated